jazz-ui/Jenkinsfile

```diff
@@ -0,0 +1,107 @@
+#!groovy
+import groovy.json.JsonOutput
+import groovy.json.JsonSlurper
+import groovy.transform.Field
+
+// To be replaced as @Field def repo_credential_id = "value" except for config_loader. env_name_prefix, jazz_bucket_web to be taken from the config_loader.
+@Field def repo_credential_id
+@Field def repo_base
+@Field def repo_core
+@Field def config_loader
+@Field def env_name_prefix
+@Field def jazz_bucket_web
+
+/*
+* Load environment variables from build module
+*/
+def load_config_module(build_module_url){
+  echo "loading environment variables, checking repos..."
+
+  dir('config-loader') {
```

Comment:
this file was merged into v1.1.3 as part of another PR correct. Did you already do a pull and merge code?

Reply:
Yes, indeed this file is already merged at https://github.com/tmobile/jazz/blob/v1.1.3/jazz-ui/Jenkinsfile. The reason why it didn't pop up in Jenkins SLF jobs is because the filename is "Jenkinsfile". The SLF monitoring job monitors only the files named "Jenkinsfile_Platform" to differentiate the platform services. We will need to rename this file too.